### DISCUSSION TITLE
質問テンプレート

### DISCUSSION THREAD

**Discussion by yukiny0811** [lit-iphone-course/lit-iphone-course#11] [お知らせ]
〇〇が動きません！こんなエラーが起きています！
〇〇は試してみましたがわかりませんでした。

![Screenshot by Dropbox Capture](https://user-images.githubusercontent.com/28947703/188383323-1e487f66-c513-4ad1-98a2-154bc024ef46.png)

Githubのリンク
https://github.com/yuruyuru-learning/quiz

**Comment by appleEmily** [comment]
この記事とか参考にしてみて〜
https: // 〇〇

**Comment by txxxxc** [comment]
最新版はこちら↓
https://github.com/lit-iphone-course/lit-iphone-course/discussions/18
